assets/dev/js/editor/container/container.js

```diff
@@ -286,28 +286,43 @@ export default class Container extends ArgsObject {
 		}
 	}
 
-	addRepeaterItem( repeaterName, rowSettingsModel, index ) {
-		let rowId = rowSettingsModel.get( '_id' );
+	/**
```

Comment:
Please send only the fix without renaming. (You can make other changes in a separated commit)

Reply:
Please check now.